core/themes/basis/css/component/vertical-tabs.css

```
@@ -10,24 +10,28 @@
   border: 1px solid #bbbbbb;
   background: #ffffff;
 }
- @media (min-width: 48em) {
+@media (min-width: 48em) {
   .vertical-tabs {
     padding: 0 0 0 15em; /* LTR */
   }
-   [dir="rtl"] .vertical-tabs {
-     padding-left: 0;
-     padding-right: 15em;
+  [dir="rtl"] .vertical-tabs {
+    padding-left: 0;
+    padding-right: 15em;
```

Since we're fixing this, should this bit of code be like so instead?:

```suggestion
    padding: 0 15em 0 0;
```

---

Agreed, we're fixing the whitespace, so why not fixing the code. :+1: